website/docs/r/api_gateway_rest_api.html.markdown

```diff
@@ -26,6 +26,18 @@ The following arguments are supported:
 * `name` - (Required) The name of the REST API
 * `description` - (Optional) The description of the REST API
 * `binary_media_types` - (Optional) The list of binary media types supported by the RestApi. By default, the RestApi supports only UTF-8-encoded text payloads.
+* `body` - (Optional) An OpenAPI specification that defines the set of routes and integrations to create as part of the REST API.
+
+__Note__: If the `body` argument is provided, the OpenAPI specification will be used to configure the resources, methods and integrations for the Rest API. If this argument is provided, the following resources should not be configured manually for the same REST API, as updates may cause manual resource updates to be overwritten:
```

Comment:
> the following resources should not be configured manually

I'd rather say _should not be managed as separate ones_ as that's more in-line with the language used in the rest of our docs.